extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -992,18 +934,18 @@ function getTitle() {
 	/**
 	 * getFieldNames
 	 *
-	 * @return Array with column names
+	 * @return array Array with column names
 	 */
 	public function getFieldNames() {
-		if( !$this->mFieldNames ) {
-			$this->mFieldNames = array();
-			if( ! $this->mWikiId ) {
-				$this->mFieldNames["city_url"]      = "Wiki";
+		if ( !$this->mFieldNames ) {
+			$this->mFieldNames = [];
+			if ( ! $this->mWikiId ) {
+				$this->mFieldNames["city_url"] = "Wiki";
 			}
-			$this->mFieldNames["cl_timestamp"]  = "Changed";
-			$this->mFieldNames["cl_type"]       = "Type";
-			$this->mFieldNames["cl_user_id"]    = "Who";
-			$this->mFieldNames["cl_text"]       = "What";
+			$this->mFieldNames["cl_timestamp"] = "Changed";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "cl_timestamp" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)